.travis.yml

```diff
@@ -32,11 +32,11 @@ matrix:
   - rvm: 2.5.8
     env: TEST_SUITE=spec:debride
 bundler_args: "--no-deployment"
-before_install: source tools/ci/before_install.sh
+before_install: source bin/ci/before_install.sh
```

Comment:
I moved these to `bin/ci` per https://github.com/ManageIQ/manageiq/pull/20984#discussion_r565504159